### DISCUSSION TITLE
Anzahl Seiten Assignment IV

### DISCUSSION THREAD

**Discussion by MehmetCagferoglu** [irgroup-classrooms/dis17-2021#39] [Q&A]
Hallo zusammen,
kurz eine Frage zur Anzahl der Seiten. Gibt es da eine Vorgabe? 
Wir haben schon einige Seiten und würden gerne wissen was die gewünschte Anzahl der Seiten ist ?
Vielen Dank

Team Shit Happens

**Comment by phschaer** [comment]
Lieber Herr Cagferoglu,

gute Frage - wie immer gilt: "Form follows function", wichtig ist also dass alles Wichtige enthalten ist. 

Als Richtgröße können Sie aber die übliche Länge eines Workshop-Papers in diesen Bereichen ansehen. Die Länge bewegt sich da im Bereich ~10 Seiten, inklusive Literaturverzeichnis und Abbildungen/Tabellen. 

Alles Gute

Philipp Schaer
